packages/base/src/utils/network.ts

```diff
@@ -142,4 +146,6 @@ const chainIds: { [key in Network]: number } = {
   'mantle': 5000,
   'scroll': 534352,
   'scroll-sepolia': 534351,
+  'meld': 0x13d92e8d,
```

Comment:
why not 333000333 ? It is bigger value than supported?

Reply:
Went with hex value, Metamask doesn't support this either